src/Parser/Core/Modules/Items/TarratusKeystone.js

```diff
@@ -0,0 +1,35 @@
+import SPELLS from 'common/SPELLS';
+import ITEMS from 'common/ITEMS';
+
+import Analyzer from 'Parser/Core/Analyzer';
+import Combatants from 'Parser/Core/Modules/Combatants';
+
+class TarratusKeystone extends Analyzer {
+  static dependencies = {
+      combatants: Combatants,
+  };
+  healing = 0;
+  cast = 0;
+
+  on_initialized() {
+    this.active = this.combatants.selected.hasTrinket(ITEMS.TARRATUS_KEYSTONE.id);
+  }
+
+  on_byPlayer_heal(event) {
+    const spellId = event.ability.guid;
+
+    if (spellId === SPELLS.TARRATUS_KEYSTONE.id) {
+      this.cast++;
```

Comment:
As a code style change, can you please update this to use the format: `this.cast += 1;`

Also it looks like this variable isn't being used for anything additional. Were you planning on adding some information around the number of casts?